shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/token/generator/impl/EncryptProjectionTokenGenerator.java

```diff
@@ -118,6 +119,17 @@ private SubstitutableColumnNameToken generateSQLTokens(final ShorthandProjection
         return new SubstitutableColumnNameToken(segment.getStartIndex(), segment.getStopIndex(), projections, databaseType.getQuoteCharacter());
     }
     
+    private void processWithFirstColumnProjection(final ColumnProjectionSegment columnSegment, final List<ColumnProjection> projections) {
```

Comment:
@cheese8 Why add this special logic? Can you consider handling similar logic inside generateProjections?